### ISSUE TITLE
treeviewComponent.allItem.correctChecked() doesn't work on viewchild

### ISSUE THREAD

**Issue by talynone** [leovo2708/ngx-treeview#122]
I have a viewchild reference to the treeview control, after checking items programatically I call allItem.correctChecked() to get the all item to set to an indeterminate or checked state, but it stays unchecked.

**Comment by leovo2708** [comment]
Yes, allItem doesn't work with correctChecked() because it doesn't have children.

**Comment by talynone** [comment]
So how do I get it to an intermediate state when checking children pragmatically?

**Comment by arunaxxl** [comment]
You got the solution?
